solutions/observability/apm/tail-based-sampling.md

```diff
@@ -85,6 +85,18 @@ Policies map trace events to a sample rate. Each policy must specify a sample ra
 | APM Server binary | `sampling.tail.policies` |
 | Fleet-managed | `Policies` |
 
+### Discard On Write Failure [sampling-tail-discard-on-write-failure-ref]
+
+Defines the indexing behavior when trace events fail to be written to storage (for example, when the storage limit is reached). When set to `false`, traces bypass sampling and are always indexed, which significantly increases the indexing load. When set to `true`, traces are discarded, causing data loss which can result in broken traces. The default is `false`.
+
+Default: `false`. (bool)
+
+|                              |                                          |
+|------------------------------|------------------------------------------|
+| APM Server binary            | `sampling.tail.discard_on_write_failure` |
+| Fleet-managed (version 9.1+) | `Discard On Write Failure`               |
```

Comment:
```suggestion
| Fleet-managed {applies_to}`stack: ga 9.1` | `Discard On Write Failure`               |
```
corresponds to review comment in https://github.com/elastic/docs-content/pull/1269#discussion_r2185883848